kotlin/internal/toolchains.bzl

```diff
@@ -264,9 +260,24 @@ _kt_toolchain = rule(
     provides = [platform_common.ToolchainInfo],
 )
 
+_KT_DEFAULT_TOOLCHAIN = Label("//kotlin/internal:default_toolchain")
+
 def kt_register_toolchains():
     """This macro registers the kotlin toolchain."""
-    native.register_toolchains(absolute_target("//kotlin/internal:default_toolchain"))
+    native.register_toolchains(str(_KT_DEFAULT_TOOLCHAIN))
+
+# Evaluating the select in the context of bzl file to get its repository
+_DEBUG_SELECT = select({
+    Label("//kotlin/internal:builder_debug_trace"): ["trace"],
```

Comment:
Note that this raises the minimum required Bazel version to 5.1 since only that version [supports `Label`s as keys in `select` dicts](https://github.com/bazelbuild/bazel/pull/14755).

@comius I am not 100% certain, but thanks to canonical label literals `str(Label(...))` should work here as well while remaining compatible with Bazel < 5.1.

Reply:
Done.